accounts/templates/accounts/researcher-registration.html

```diff
@@ -13,7 +13,7 @@ <h1 class="panel-title">Create Researcher Account</h1>
                     {% csrf_token %}
                     {% bootstrap_form form %}
                     {% buttons %}
-                        <p> By clicking 'Create Account', you agree that you have read and accepted the <a href="/termsofuse/" target="_blank">Terms of Use</a> and <a href="/privacy/" target="_blank">Privacy Statement</a>.
+                        <p> By clicking 'Create Account', you agree that you have read and accepted the <a href="../termsofuse/" target="_blank">Terms of Use</a> and <a href="../privacy/" target="_blank">Privacy Statement</a>.
```

Comment:
I think I am going to do some research here, but I am still on the idea that we should be using the [URL tag](https://docs.djangoproject.com/en/3.2/ref/templates/builtins/#url) to generate urls.  Since translations (and the translations view) are handled by Django, it would seem like the urls would also be updated on HTML creation.